homeassistant/components/vallox/sensor.py

```diff
@@ -202,6 +218,7 @@ class ValloxSensorEntityDescription(SensorEntityDescription):
         icon="mdi:gauge",
         state_class=SensorStateClass.MEASUREMENT,
         native_unit_of_measurement=PERCENTAGE,
+        entity_registry_enabled_default=False,
```

Comment:
Just out of curiosity, what's the thinking behind disabling this by default?

Reply:
This "Efficiency" in my opinion is useless metric. Never seen it changing. Mine showed something around 65% while temperature inside and outside is nearly the same.
I think who wants it can enable it.

Reply:
> This "Efficiency" in my opinion is useless metric. Never seen it changing. Mine showed something around 65% while temperature inside and outside is nearly the same. I think who wants it can enable it.

I do note efficiency, but I don't think anybody needs it to three decimal places. Fan speed % and Humidity % are both integers. Efficiency % as an integer would work very well.

Reply:
@PhillyGilly Is "Efficiency" showing 65,535 % for you? It is not 65.535. It is 65535 which is 0xFFFF in hex. Which means is undefined =)

Reply:
@yozik04 The Vallox API returns all kinds of funny numbers as "not currently available". 65535 is one, and zero degrees Kelvin (-273) is another. I have occasionally thought about where to add a list of these fake values, and where to convert them to None (either the HA side or the library side).

Reply:
Of course it needs to be in the library in the end...

Reply:
@viiru- I need precise value for zero degrees Kelvin. Is it exactly -273 or has some decimals?

Reply:
@yozik04 To me Vallox().fetch_metrics() returns stuff like this:
```
 'A_CYC_OPT_TEMP_SENSOR_MODE': -273.1,
 'C_CYC_OPT_TEMP_EXTRACT': -273.1,
 'C_CYC_OPT_TEMP_MLV': -273.1,
 'C_CYC_OPT_TEMP_NONE': -273.1,
 'EXT_AWAY_AIR_TEMP_TARGET': -273.1,
 'EXT_BOOST_AIR_TEMP_TARGET': -273.1,
 'EXT_CYC_DEFROST_TEMP_PARAM': -273.1,
 'EXT_EXTRA_AIR_TEMP_TARGET': -273.1,
 'EXT_HOME_AIR_TEMP_TARGET': -273.1,
 'EXT_TEMP_NOW_EXHAUST_AIR': -273.1,
 'EXT_TEMP_NOW_EXTRACT_AIR': -273.1,
 'EXT_TEMP_NOW_OUTDOOR_AIR': -273.1,
 'EXT_TEMP_NOW_SUPPLY_AIR': -273.1,
```

Reply:
Great! Thank you. Will fix in lib.

Reply:
Ok. I fixed it in the library and bumped the library as well.